Makefile

```diff
@@ -39,15 +36,14 @@ endif
 ifneq ($(NO_CACHE),)
 	DOCKER_BUILD_ARGS += --no-cache
 endif
-WIN_BUILD_ARGS := DOCKER_BUILD_ARGS
 DOCKER_BUILD_ARGS += --build-arg OVS_VERSION=$(OVS_VERSION)
 DOCKER_BUILD_ARGS += --build-arg GO_VERSION=$(GO_VERSION)
 DOCKER_BUILD_ARGS += --build-arg BUILD_TAG=$(BUILD_TAG)
-WIN_BUILD_ARGS := --build-arg GO_VERSION=$(GO_VERSION)
-WIN_BUILD_ARGS += --build-arg CNI_BINARIES_VERSION=$(CNI_BINARIES_VERSION)
-WIN_BUILD_ARGS += --build-arg NANOSERVER_VERSION=$(NANOSERVER_VERSION)
-WIN_BUILD_ARGS += --build-arg WIN_BUILD_TAG=$(WIN_BUILD_TAG)
-WIN_BUILD_ARGS += --build-arg WIN_BUILD_OVS_TAG=$(WIN_BUILD_OVS_TAG)
+
+WINDOWS_BUILD_ARGS :=
+ifneq ($(WINDOWS_PUSH),)
```

Comment:
If we could avoid introducing this, it would be good IMO. It seems that this is only used for the `build-windows` target, and existing behavior of the target is that image is build but not pushed.

Reply:
As I planned to use docker buildx to build Windows image on Linux VM, the built image can be either saved in a local path by specifying `type=docker,dest=$local_path`,  or pushed to the target registry  by specifying `type=registry`. It is finished by one docker command. If we don't specify the output type in docker buildx build, we can't find the image from local registry later as it is build with multi-platform. 
So it requires that before executing the final build command, we shall collect the requirements to push or save locally. 

For CI use cases and dev environement, we could save the image on the Linux VM where building the image, it is the default behavior for the Makefile. For the github daily push job,it shall tell the intent to push the image finally. If we don't introduce any flag in Makefile. then we can't get the desired behavior, so it only save the file locally. @antoninbas any suggestions?

Reply:
An alternative in my mind is, I add the consumer of "WINDOWS_PUSH" in build-windows.sh directly, and remove the consumption from Makefile. That is to say, both "./build-windows.sh --push" or "WINDOWS_PUSH=1; ./build-windows.sh" mean that a push is desired. What do you think?